src/core/three/EffectComposer.vue

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+import { useLoop, useTresContext } from '@tresjs/core'
+import { useDevicePixelRatio } from '@vueuse/core'
+import { EffectComposer as EffectComposerThreejs } from 'three/examples/jsm/postprocessing/EffectComposer.js'
```

Comment:
Hi @Tinoooo change all the `three/examples/jsm` imports to `three-stdlib` which is a stand-alone version of [threejs/examples/jsm](https://github.com/mrdoob/three.js/tree/dev/examples/jsm) written in Typescript & built for ESM & CJS. This way we ensure proper tree-shaking

Reply:
I would like to use `three-stdlib`, but their EffectComposer work differently. It does not have a dispose method, which is important for memory stuff.